Makefile

```diff
@@ -200,11 +200,11 @@ test-libponyc: all
 
 test-full-programs-release: all
 	@mkdir -p $(outDir)/runner-tests/release
-	$(SILENT)cd '$(outDir)' && $(buildDir)/test/libponyc-run/runner/runner --sequential=true --exclude=runner --ponyc=$(outDir)/ponyc --output=$(outDir)/runner-tests/release --test_lib=$(outDir)/test_lib $(srcDir)/test/libponyc-run
+	$(SILENT)cd '$(outDir)' && $(buildDir)/test/libponyc-run/runner/runner --timeout_s=60 --sequential=false --max_parallel=`nproc --all` --exclude=runner --ponyc=$(outDir)/ponyc --output=$(outDir)/runner-tests/release --test_lib=$(outDir)/test_lib $(srcDir)/test/libponyc-run
```

Comment:
can we remove --sequential entirely? i think we can, yes?

Reply:
Yes, it defaults to `false`.  I'll remove it.